### PR TITLE
added import and snapshot name to Optional

### PR DESCRIPTION
**Pull request by Abhishekism9450**
Imports for restore, scale, tags, cluster, maintenance window, profile, slas.
Snapshot resource : name is set to Optional with default. 
d.id is being checked for nil in readContext.

### CODE REVIEW

**Review by siddharth-kulshrestha — APPROVED** [review]

**Review by siddharth-kulshrestha — APPROVED** [review]

File: nutanix/data_source_nutanix_ndb_profiles_available_ips_test.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+func TestAccEraProfilesAvailableIPsDataSource_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccEraPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccEraProfilesAvailableIPsDataSourceConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.nutanix_ndb_network_available_ips.ips", "available_ips.#"),
+					resource.TestCheckResourceAttrSet("data.nutanix_ndb_network_available_ips.ips", "available_ips.0.id"),
```
Comment (bhati-pradeep) [inline nutanix/data_source_nutanix_ndb_profiles_available_ips_test.go]: can we check if multiple values are coming or not

File: nutanix/resource_nutanix_nbd_database_test.go [hunk 2]
```diff
@@ -27,6 +27,7 @@ func TestAccEra_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceNameDB, "databasetype", "postgres_database"),
 					resource.TestCheckResourceAttr(resourceNameDB, "database_nodes.#", "1"),
 					resource.TestCheckResourceAttrSet(resourceNameDB, "time_machine_id"),
+					resource.TestCheckResourceAttrSet(resourceNameDB, "time_machine.#"),
```
Comment (bhati-pradeep) [inline nutanix/resource_nutanix_nbd_database_test.go:30]: In file name it should be "_ndb_"

**Review by bhati-pradeep — CHANGES_REQUESTED** [review]

File: nutanix/data_source_nutanix_ndb_profiles_available_ips_test.go (hunk 1 above)
Comment (Abhishekism9450) [reply nutanix/data_source_nutanix_ndb_profiles_available_ips_test.go]: Checked in PR #570 .

**Review by bhati-pradeep — APPROVED** [review]
LGTM

**Review by abhimutant — APPROVED** [review]

**Pull request merged**